### PR TITLE
coment class/route/controller

### PR DESCRIPTION
**Pull request by Kify7**

### CODE REVIEW

**Comment by Kify7** [comment]
Adding coments

**Review by rachelldgz — APPROVED** [review]

File: Routes/coments.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const router = require('express').Router();
+const {
+    createComent,
```
Comment (rachelldgz) [inline Routes/coments.js:3]:
```suggestion
    createComment,
```

File: Routes/coments.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    getComent,
```
Comment (rachelldgz) [inline Routes/coments.js:4]:
```suggestion
    getComment,
```

File: Routes/coments.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    updateComent,
```
Comment (rachelldgz) [inline Routes/coments.js:5]:
```suggestion
    updateComment,
```

File: Routes/coments.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    deleteComent
```
Comment (rachelldgz) [inline Routes/coments.js:6]:
```suggestion
    deleteComment
```

File: Routes/coments.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+} = require('../controllers/coments')
+
+router.post('/', createComent)
```
Comment (rachelldgz) [inline Routes/coments.js:9]: router.post('/', createComment)

**Pull request merged**